locales/de/LC_MESSAGES/development.po

```diff
@@ -117,103 +101,85 @@ msgstr ""
 "einzuschließen oder führen Sie ihn mit dem Fixup-Commit vor dem Pushen "
 "zusammen."
 
-#: /home/user/w/MultiGeiger/docs/source/development.rst:37
-#: c746e2c4835a4d3cb70041228d9c8174
+#: ../../source/development.rst:37 c746e2c4835a4d3cb70041228d9c8174
 msgid "have a nice, clear, typo-free commit comment"
 msgstr ""
 "schreiben Sie einen schönen, klaren, tippfehlerfreien Commit-Kommentar"
 
-#: /home/user/w/MultiGeiger/docs/source/development.rst:38
-#: f1b869d0e75149af8d2b0252d5452e04
+#: ../../source/development.rst:38 f1b869d0e75149af8d2b0252d5452e04
 msgid "if you fixed an issue, refer to it in your commit comment"
 msgstr ""
 "wenn Sie ein Problem behoben haben, verweisen Sie darauf in Ihrem Commit-"
 "Kommentar"
 
-#: /home/user/w/MultiGeiger/docs/source/development.rst:40
-#: c710303ab2a34a04b9bde686a9bda789
+#: ../../source/development.rst:40 c710303ab2a34a04b9bde686a9bda789
 msgid ""
-"make a pull request on github and check on the PR page what ``travis-ci`` "
+"make a pull request on github and check on the PR page what the CI system "
 "tells about the code in your PR"
 msgstr ""
-"Machen Sie einen Pull-Request auf Github und überprüfen Sie auf der PR-"
-"Seite, was ``travis-ci`` über den Code in Ihrem PR sagt"
```

Comment:
ah, likely this got "lost" (became fuzzy) because i changed the original string.